dessia_common/core.py

```diff
@@ -874,7 +874,7 @@ def get_booleans_index(self, values: List[DessiaObject]):
         :param values: List of DessiaObjects to filter
         :type values: List[DessiaObject]
 
-        :return: `list of length `len(values)` where elements are `True` if kept by the filter, otherwise `False`.
+        :return: `list of length `len(values)` where the elements are `True` if kept by the filter, otherwise `False`.
```

Comment:
I don't think docstring should be changed here; Old version seem more "english" to me